cluster/node-pools/master-default/userdata.yaml

```diff
@@ -745,6 +750,24 @@ write_files:
           omitStages:
             - "RequestReceived"
 
+{{ if and (eq .Cluster.ConfigItems.enable_default_sa "true") (eq .Cluster.ConfigItems.auditlog_read_access "true") }}
+  - owner: root:root
+    path: /etc/kubernetes/config/audit-policy-ro.yaml
+    content: |
+      apiVersion: audit.k8s.io/v1beta1
+      kind: Policy
+      rules:
+      - level: None
+        verbs: ["create", "update", "patch", "delete", "deletecollection"]
```

Comment:
I understand why the following `None` rule is necessary:
```
- level: None
   userGroups: ["system:serviceaccounts:kube-system"]
```
Could you say why we need to explicitly disable the `verbs` we nevery enabled?

Reply:
Good point, this is not actually needed. I have dropped it.